src/layout/Sidebar.tsx

```diff
@@ -63,8 +57,30 @@ const Sidebar = () => {
     const storedValue = localStorage.getItem('sidebarOpen');
     return storedValue == null ? true : JSON.parse(storedValue);
   });
+  const arioProcessId = useGlobalState((state) => state.arioProcessId);
 
   const [showChangLogModal, setShowChangeLogModal] = useState(false);
+  const [showSettingsModal, setShowSettingsModal] = useState(false);
+
+  const ROUTES_SECONDARY = [
+    {
+      title: 'Docs',
+      icon: <DocsIcon className="size-4" />,
+      path: ARIO_DOCS_URL,
+    },
+    {
+      title: 'Process',
+      icon: <ContractIcon className="size-4" />,
+      path: `https://www.ao.link/#/entity/${arioProcessId}`,
+    },
+    {
+      title: 'Settings',
+      icon: <Settings className="size-4" />,
```

Comment:
note: should test on mobile, lucide can be finicky with setting an icon as a variable like this. We can wrap it if thats the case

Reply:
Network Portal is not designed for mobile at the moment, but good point, will keep this in mind when we add mobile support.